### ISSUE TITLE
Varnish Cache and Invalidation is not working

### ISSUE THREAD

**Issue by iCodr8** [api-platform/api-platform#1050]
The documentation describes that varnish is working out of the box.
https://api-platform.com/docs/core/performance/

But it doesn't.

I am using `localhost:8081` and the GET requests are not faster, when I call the same endpoint multiple times. I thought that when I call the endpoint the first time, that it will be cached and on the next (same) request, I will get the data from the cache.

So I tried something. I have set the `max_age` to `60` and called the endpoint multiple times. The first was slow and the other were super fast (20-40ms). And now I could see in the header, that the data was from the cache. But the problem was that the invalidation was not working and I got the new data only after the 60 seconds.

First I thought this is only a problem on my application, but then I have tested it on https://demo.api-platform.com and there it is also not working.

@dunglas: Do I understand something wrong or does it really not work?

**Comment by iCodr8** [comment]
Api Platform Version 2.3.6

**Comment by ilyakonstantinov95** [comment]
Same problem with v2.4.2

**Comment by teohhanhui** [comment]
We'd need more details to understand the problem you're facing.

**Comment by ilyakonstantinov95** [comment]
Requests are not responing via varnish cash in dev mode

**Comment by addinm** [comment]
+++

Any update on this?

**Comment by domgraziano** [comment]
confirming following the docs and trying to speeding up the api platform, I can't see any varnish container when running`docker ps`

**Comment by soyuka** [comment]
Closing per #2987 — this repository is becoming a CLI installer. The previous Docker/template distribution is being superseded by the installer + upstream [symfony-docker](https://github.com/dunglas/symfony-docker). If the issue still reproduces with the new setup, please open it on `dunglas/symfony-docker` (for Docker/Caddy/FrankenPHP issues) or `api-platform/core` (for framework issues).
